### PR TITLE
Update icon svg

### PR DESCRIPTION
**Pull request by runeb**
Updated to latest icon found at https://collection-share.cloudinary.com/cloudinary-marketing/a42272e5732d30dc77f6b1d06c8f4dda

Screen Shot 2021-03-11 at 4 06 13 PM

### CODE REVIEW

**Review by skogsmaskin — APPROVED** [review]
Thanks @runeb!

Let's wait for the Sanity UI team's feedback on the alignment of the icon before merging.

**Comment by skogsmaskin** [comment]
@runeb - did you find out anything about the icon alignment?

**Comment by runeb** [comment]
@skogsmaskin No, assuming its a bug in @sanity/ui ?

**Pull request closed without merging**
